README.md

```diff
@@ -27,6 +27,63 @@ Privoxy exposes an HTTP proxy.
 - `TORS` — Number of Tor instances (default: 5)
 - `HAPROXY_LOGIN` — Username for HAProxy (default: "admin")
 - `HAPROXY_PASSWORD` — Password for HAProxy (default: "admin")
+- `TOR_BRIDGES` - Bridge multiline string with bridges records (default : "")
+- `TOR_EXIT_NODES` - Tor exit nodes config (default : "" , for example `TOR_EXIT_NODES=ru` or `TOR_EXIT_NODES=ru,en`)
+
+
+Note: If `TOR_BRIDGES` is empty bgidges feature will disabled
+
+## Tor Bridges Configuration
+
+To enable the Bridges function, you must specify the bridges using one of the following variations (to reduce priority):
+1. in the `bridges.lst` file
+
+    ```
+    Bridge obfs4 37.18.133.75:44821 D40DA77CA68F39666F77CE8BA6FF332BF8DB3F31 cert=B4yVW8heE83luCJt+oQN1kDB/j4kWkNx6mtOc9O6GhLAV8zJx0lfUI6zWO9UxUoV5PX/Zw iat-mode=0
+    Bridge obfs4 51.81.26.157:443 8A7322A463C051DB6DC35B1159F119FC3373BB06 cert=kp6Czj/f+McG9OKwltQ4kGb41mjj8Mzp3flpTG8/VK5zXtfnZ+DToe33fumyq7Yq7WnbGA iat-mode=0
+    ```
+
+2. in the `TOR_BRIDGES` environment variable in the `docker` command call
+
+    ```bash
+    docker run --rm --name medusa-proxy -e TORS=3 -e HEADS=2 \
+        -e TOR_USE_BRIDGES=1
```

Comment:
Is this environment variable used still? It seems that we are now inferring this from whether bridges are specified. Thanks for making that change BTW.